### DISCUSSION TITLE
GitHub device authorization page not opening from VS 2022

### DISCUSSION THREAD

**Discussion by pradeepviswav** [community/community#13986] [Copilot Conversations]
GitHub Copilot is already enabled for my account. After I installed the Copilot add-in in VS 2022, a message to activate the connection to GitHub appears. In general, the GitHub device authorization page will open where I can paste the activation code. In my case, browser window is not opened. How do I manually open GitHub device auth page to paste my activation code?

**Comment by kzu** [comment]
If you're like me, it's likely that the page got lost within 42 other tabs on an already opened default browser window :). You can navigate manually to https://github.com/login/device and paste the code there. 👍

**Reply by juancarlosortizr** [reply]
This is not helpful at all. Where is the code? I go there and if I paste, I get nothing.

**Comment by ricnader** [comment]
No code on clipboard :/

**Comment by JWaslo** [comment]
Has anyone found a workaround for this issue?
